integration/src/test/java/org/apache/iotdb/db/integration/IoTDBMetadataFetchIT.java

```diff
@@ -52,7 +52,6 @@
  * Notice that, all test begins with "IoTDB" is integration test. All test which will start the
  * IoTDB server should be defined as integration test.
  */
-@Category({LocalStandaloneTest.class, ClusterTest.class, RemoteTest.class})
```

Comment:
The test case `databaseMetaDataTest()` needs to dump the whole MTree as a json string. However in a cluster, each node doesn't have the whole tree. And I think the metadata interface in JDBC is not called regularly.